### PR TITLE
Add packaging condition to format exception message correctly in MavenProjectProperties#createJibContainerBuilder

### PR DESCRIPTION
**Pull request by JacekDuszenko**
fixes: #3214
Added packaging condition to format exception message correctly

### CODE REVIEW

**Review by elharo — APPROVED** [review]

**Comment by JacekDuszenko** [comment]
Thanks for approval @elharo !

File: jib-maven-plugin/src/test/java/com/google/cloud/tools/jib/maven/MavenProjectPropertiesTest.java [hunk 1]
```diff
@@ -570,6 +573,49 @@ public void testCreateContainerBuilder_noErrorIfWebInfLibDoesNotExist()
...
+    String expectedMessage =
+        "Obtaining project build output files failed; make sure you have "
+            + "compiled your project before trying to build the image. (Did you accidentally run \"mvn clean "
+            + "jib:build\" instead of \"mvn clean compile jib:build\"?)";
+
+    when(mockMavenProject.getPackaging()).thenReturn("jar");
+    when(mockJavaContainerBuilder.addResources(any(), any())).thenThrow(IOException.class);
+
+    IOException thrownException =
+        Assert.assertThrows(
```
Comment (chanseokoh) [inline jib-maven-plugin/src/test/java/com/google/cloud/tools/jib/maven/MavenProjectPropertiesTest.java:611]:
```suggestion
        assertThrows(
```

File: jib-maven-plugin/src/test/java/com/google/cloud/tools/jib/maven/MavenProjectPropertiesTest.java [hunk 2]
```diff
@@ -570,6 +573,49 @@ public void testCreateContainerBuilder_noErrorIfWebInfLibDoesNotExist()
...
+            IOException.class,
+            () ->
+                mavenProjectProperties.createJibContainerBuilder(
+                    mockJavaContainerBuilder, ContainerizingMode.EXPLODED));
+    Assert.assertEquals(thrownException.getMessage(), expectedMessage);
```
Comment (chanseokoh) [inline jib-maven-plugin/src/test/java/com/google/cloud/tools/jib/maven/MavenProjectPropertiesTest.java:616]:
```suggestion
    assertThat(thrownException).hasMessageThat().isEqualTo(expectedMessage);
```

**Review by chanseokoh — APPROVED** [review]
LGTM! Very minor style suggestions.

**Comment by JacekDuszenko** [comment]
@chanseokoh  @elharo fixed minor style issues. This one is I believe available to be merged now :)

**Comment by chanseokoh** [comment]
@JacekDuszenko thank you for your time to fix this! Merging now.

**Pull request merged**